.azure-pipelines/templates/azdev_setup.yml

```diff
@@ -14,7 +14,7 @@ steps:
       chmod +x env/bin/activate
       . env/bin/activate
 
-      pip install -q azdev
+      pip install -q git+https://github.com/Azure/azure-cli-dev-tools.git@master
```

Comment:
do we need use the master branch? Since azdev is our internal dev tool, it won't bring too much pros for me to use latest master branch.

Reply:
@jiasli comment here https://github.com/Azure/azure-cli-dev-tools/pull/220#issuecomment-652772354.
Using the master branch could bring us a sensitive and quick sense on the change of azdev changes without release it.
I think currently, release of azdev is too much frequently.

Reply:
We should figure out better way to handle this. azdev release process should be improved also.